fitsio/util.py

```diff
@@ -41,7 +41,7 @@ def cfitsio_version(asfloat=False):
             numpy.uint64, numpy.int64)
 
 # different for py3
-_stypes += (numpy.string_, numpy.str_)
+_stypes += (numpy.str_, )
```

Comment:
We should preserve the use of `numpy.string_` for <2.

Reply:
Actually the migration guide says to replace `np.string_` with `np.bytes_` so I think we need that. https://numpy.org/devdocs/numpy_2_0_migration_guide.html#main-namespace